credentials/sts/sts_test.go

```diff
@@ -441,7 +419,7 @@ func (s) TestGetRequestMetadataBadSubjectTokenRead(t *testing.T) {
 		ctx, cancel := context.WithTimeout(context.Background(), defaultTestTimeout)
```

Comment:
...so then THIS deadline should be short, since you're expecting a timeout.  The problem is you are re-using `defaultTestTimeout` for both positive and negative tests.

Reply:
Done.